integrationTests/oats/build.gradle

```diff
@@ -0,0 +1,15 @@
+plugins {
```

Comment:
I'm not sure what is oats or why this is set-up this way but you can put the integration tests into the `test` sourceset of the component that they are testing, i.e.: `/src/test` or into a new sourceset: `/src/integrationTest`.

Reply:
1. oats: https://github.com/grafana/oats/tree/main/yaml
2. .. in the directory: I couldn't get that to work :disappointed: